### PR TITLE
Removed excess buttons in entry overview, replacing it with button th…

### PR DESCRIPTION
**Pull request by IR96334**
…at releases a dropdown menu

### CODE REVIEW

File: frontend/src/entry/overview/FormEditPage.tsx [hunk 1]
```diff
@@ -137,28 +148,30 @@ export default function FormEditPage({ entry, currentUserRoles, readOnly = false
             </Stack>
           </div>
           {!isEdit && (
-            <Stack direction='row' spacing={1} justifyContent='flex-end' sx={{ mb: { xs: 2 } }}>
-              <Button variant='outlined' onClick={() => setExportDialogOpen(true)}>
-                Export as PDF
-              </Button>
-              <Button variant='outlined' onClick={() => setHistoryDialogOpen(true)}>
-                View History
+            <div>
```
Comment (ARADDCC012) [inline frontend/src/entry/overview/FormEditPage.tsx:151]: This can be a fragment

File: frontend/src/entry/overview/FormEditPage.tsx [hunk 2]
```diff
@@ -137,28 +148,30 @@ export default function FormEditPage({ entry, currentUserRoles, readOnly = false
...
-                    >
-                      {`Edit ${EntryCardKindLabel[entry.kind]}`}
-                    </Button>
-                  </span>
-                </Tooltip>
-              )}
-            </Stack>
+              <Menu anchorEl={anchorEl} open={open} onClose={handleActionButtonClose}>
+                <MenuItem>
+                  <ListItemText onClick={() => setExportDialogOpen(true)}>Export as PDF</ListItemText>
```
Comment (ARADDCC012) [inline frontend/src/entry/overview/FormEditPage.tsx:157]: We also want to close the menu when this is clicked

File: frontend/src/entry/overview/FormEditPage.tsx [hunk 3]
```diff
@@ -137,28 +148,30 @@ export default function FormEditPage({ entry, currentUserRoles, readOnly = false
...
+                </MenuItem>
+                <MenuItem>
+                  <ListItemText onClick={() => setHistoryDialogOpen(true)}>View History</ListItemText>
```
Comment (ARADDCC012) [inline frontend/src/entry/overview/FormEditPage.tsx:160]: We also want to close the menu when this is clicked

File: frontend/src/entry/overview/FormEditPage.tsx [hunk 4]
```diff
@@ -137,28 +148,30 @@ export default function FormEditPage({ entry, currentUserRoles, readOnly = false
...
+                </MenuItem>
+                {!readOnly && (
+                  <Tooltip title={requiredRolesText}>
+                    <span>
+                      <MenuItem disabled={!canEdit}>
+                        <ListItemText onClick={() => setIsEdit(!isEdit)} data-test='editEntryCardButton'>
```
Comment (ARADDCC012) [inline frontend/src/entry/overview/FormEditPage.tsx:166]: We also want to close the menu when this is clicked

**Review by ARADDCC012 — CHANGES_REQUESTED** [review]

**Comment by ARADDCC002** [comment]
I think it would be nice if we added chevrons to the button so that it points down when closed:

image

And then like so when it's opened:

image

I also think the list would like nicer with icons on the left-hand side:

image

The above example is also using the `dense` prop which removes some of the padding which I think looks a bit nicer.

File: frontend/src/entry/overview/FormEditPage.tsx [hunk 5]
```diff
@@ -137,28 +148,50 @@ export default function FormEditPage({ entry, currentUserRoles, readOnly = false
             </Stack>
           </div>
           {!isEdit && (
-            <Stack direction='row' spacing={1} justifyContent='flex-end' sx={{ mb: { xs: 2 } }}>
-              <Button variant='outlined' onClick={() => setExportDialogOpen(true)}>
-                Export as PDF
-              </Button>
-              <Button variant='outlined' onClick={() => setHistoryDialogOpen(true)}>
-                View History
+            <Fragment>
```
Comment (ARADDCC002) [inline frontend/src/entry/overview/FormEditPage.tsx:151]: `<Fragment></Fragment>` can be replaced with `<></>`

File: frontend/src/entry/overview/FormEditPage.tsx [hunk 6]
```diff
@@ -137,28 +148,50 @@ export default function FormEditPage({ entry, currentUserRoles, readOnly = false
...
+                      handleActionButtonClose()
+                      setHistoryDialogOpen(true)
+                    }}
+                  >
+                    View History
+                  </ListItemText>
+                </MenuItem>
+                {!readOnly && (
+                  <Tooltip title={requiredRolesText}>
+                    <span>
```
Comment (ARADDCC002) [inline frontend/src/entry/overview/FormEditPage.tsx:178]: I don't think we need to wrap the `<MenuItem />` in a `<span />`

**Review by ARADDCC002 — CHANGES_REQUESTED** [review]

**Review by ARADDCC012 — APPROVED** [review]
Approved pending approval from @ARADDCC002

**Review by ARADDCC002 — APPROVED** [review]
Approved - but we should wait for #1455 to go in first, then merge those changes into this PR.

**Review by ARADDCC002 — CHANGES_REQUESTED** [review]
Sorry reverting previous approval as we're still waiting on some changes.

File: frontend/src/entry/overview/FormEditPage.tsx (hunk 6 above)
Comment (IR96334) [reply frontend/src/entry/overview/FormEditPage.tsx:178]: Removed

**Review by ARADDCC002 — APPROVED** [review]

**Pull request merged**